### ISSUE TITLE
Image moves when typing long title

### ISSUE THREAD

**Issue by Lwoodruby** [ViDA-NYU/ARIES-Issues#5]
Hi - I've put in a long title, and as I type, the image moves on the image menu!  Very funny.

Also, would it be possible for the dimensions boxes to be bigger?  You can't see the whole number after you've typed it - ie 135 shows as 13  etc.

I'm still having trouble with loading two images at once.  I left my computer running for an hour and it still wouldn't let me close the upload box.  The internet here seems to be fairly good, so...???

We might want to put the "clear canvas" button somewhere else.  It took me awhile to figure out that it was under "Adjust".  But I can't think where!  There are already a lot of buttons....

**Comment by joaorulff** [comment]
Hello, Louisa. Thanks for the feedback.

We notice these bugs last week, but we already fixed it.
We made a new version available yesterday. I just tried to upload 10 images at once from my computer and worked. Did you notice these bugs today?

Regarding the dimensions field, It should be available within 30 minutes. If you're still having problems, let me know.

**Comment by Lwoodruby** [comment]
Hi Joao:  I see the dimensions are fixed - yay!  

Yes, I had the upload problem today.  I'll try again.  

I'm also still having the problem of the images moving as I type in metadata.  

Another trouble I'm having is with the "crop" function.  I highlight the image then click outside it, but nothing happens.  Am I doing something wrong?  Thanks so much!!!!

**Comment by Lhaylla** [comment]
What are the sizes of the images you are trying to upload in MB?

**Comment by joaorulff** [comment]
@Lwoodruby you're right, we had a problem with the crop function, but now it seems to be working. I just fixed it.

About the metadata problem: we're not having this issue at all at the moment. But we'll keep trying to find if there is any error. Sometimes, when we make a new version available online it takes a little longer for you to see the changes, maybe this is happening now.

About the image upload: it seems to be working, can you please send us the images you're trying to upload?

**Comment by Lwoodruby** [comment]
Here are two images that seem to load, but then i'm unable to close the window (see screenshot).  Its really bad with bigger images.


The crop works now.  Yay!!!!


Sorry - two more issues, having to do with opening Aries.  Let's say I get the screen shot above.  In order to get out of this screen, I need to hit the refresh button to start ARIES anew.  At first, nothing happens.  I have to refresh twice to get the login screen.  This has always happened.


Then, when I go to open a project, if there are lots of images, it takes quite a while for it to open. Is this normal?


I hate to be such a bother!  But its probably good I find these glitches rather than someone else!


The thing is, I find them because I"M REALLY ENJOYING AND USING THE TOOL FOR MY RESEARCH!!!  I Love it!!  :)

________________________________

**Comment by joaorulff** [comment]
Relax, @Lwoodruby. This is normal when we're developing a new system. These problems will always exist. Every software development team has its own maintenance team to deal with this kind of issues over time. =))

I can't see the screenshots you sent. You can send them by email.

About loading time, it will depend on your internet connection. I can try to perform some kind of evaluation regarding the loading time and send them to you. 

About refreshing the page: You're actually not supposed to refresh the page and go back to the same page you were before, otherwise the login system would make no sense. It's happening because you're trying to refresh 'https://artimageexplorationspace.com/home', and you can only access this page if you first login into the system. If you really need to refresh the page, you need to go to 'https://artimageexplorationspace.com'. 

Good to hear that you're enjoying using ARIES. =)
